config/initializers/access_control.rb

```diff
@@ -0,0 +1,2 @@
+yaml = YAML.load_file("#{Rails.root}/config/access_control.yml") rescue { }
```

Comment:
Rails 4.2 introduced a special method to read config files: `Rails.application.config_for(:access_control)`
If you want to use that (which is your call, 'cuz I know you're frying some big big fish), looks like you'd need to add per-environment settings to `config/access_control.yml` (otherwise `config_for` returns an empty hash).